docs/about/overview/radio-settings.mdx

```diff
@@ -25,16 +25,23 @@ Power limits will generally be lifted in the software if `is_licensed` is set to
 
 ## Europe Frequency Bands
 
+EU countries will generally be covered by the Radio Equipment Directive ([2014/53/EU](http://data.europa.eu/eli/dir/2014/53/2023-10-01)) and subsequent amendments, which is then enacted in local laws. Useful information on this is located [here](https://single-market-economy.ec.europa.eu/single-market/european-standards/harmonised-standards/radio-equipment_en).
+The bands below were selected based on the maximum power and the duty cycle available. It may be possible to use other frequency bands for specific applications, but these are not currently supported "out of the box".
+
 ### 433 MHz
 
 The maximum power allowed for Europe is +10 dBm ERP ([Effective Radiated Power](https://en.wikipedia.org/wiki/Effective_radiated_power)).
 
 The band range is from 433 to 434 MHz.
 
-There are four frequendy slots defined with the standard radio preset `LongFast`. After factory reset the radio will be set to frequency slot 4 with a center frequency of 433.875 MHz.
+There are four frequency slots defined with the standard radio preset `LongFast`. After factory reset the radio will be set to frequency slot 4 with a center frequency of 433.875 MHz.
 
 ### 868 MHz
 
+:::info
```

Comment:
I really don't feel this needs to be an admonition, it can just be regular text like the rest of the section

Reply:
I disagree that this is an admonition - it's not a warning, it's an informative note. If I wasn't putting it in an info box, I'd be starting the sentence "It is worth noting that..."

Additionally, this particular paragraph is different than the other information presented, and is intended for slightly more casual users. Highlighting it feels like the right thing to do, given that there are a few people who didn't know this and have bought low-frequency devices as a result.

Happy to be overruled if the above doesn't persuade, but I do want to make a case!

Reply:
Ok, thanks for that. I learned something new today!

To the point: is the information sufficiently different than the other content to justify placing it more prominently?

Reply:
Understood. I've moved the sentence to the bottom, and tidied up. I'm marking this as resolved.